docs/src/references/mqtt-api.md

```diff
@@ -580,7 +581,6 @@ The following table details some example command types which are supported by %%
 | firmware_update | `te/<identifier>/cmd/firmware_update/<cmd_id>` |
 | restart         | `te/<identifier>/cmd/restart/<cmd_id>`         |
 | log_upload      | `te/<identifier>/cmd/log_upload/<cmd_id>`      |
-| health          | `te/<identifier>/cmd/health/check`             |
```

Comment:
Need to restore that, right? Since we didn't commit to changing this now to avoid breaking backward compatibility.

Reply:
Good catch. Reverted in f593c4bc51f62d7ab40d6899770ff856381ead93